rotkehlchen/tests/fixtures/globaldb.py

```diff
@@ -74,6 +74,12 @@ def fixture_remove_global_assets() -> list[str]:
     return []
 
 
+@pytest.fixture(name='load_global_caches')
+def fixture_load_global_caches() -> list[str]:
+    """The decoder for which the global cache should be loaded."""
```

Comment:
Can be multiple right?

```suggestion
    """The decoders for which the global cache should be loaded."""
```

Reply:
Can you also document here that what we use is the name of the folder for the decoder? Do you think it would make sense to use the counterparty? I believe is less error prone. Instead of checking for `in` when you patch you will add a new column for the counterparty